static/usage/v7/action-sheet/inline/trigger/javascript.md

```diff
@@ -0,0 +1,34 @@
+```html
+<ion-button id="open-action-sheet">Open</ion-button>
+<ion-action-sheet
+  trigger="open-action-sheet"
+  header="Actions"
+></ion-action-sheet>
+
+<script>
+  const actionSheet = document.querySelector('ion-action-sheet');
```

Comment:
cc @sean-perkins does this need to be `var` still or has the StackBlitz bug been fixed?

Reply:
Went ahead and changed it to `var` just in case.